arch/risc-v/src/mpfs/mpfs_ihc.c

```diff
@@ -189,6 +202,11 @@ static uint16_t g_vq_idx;
 static int      g_plic_irq;
 static bool     g_rptun_initialized;
 
+#ifdef IHC_AVOID_ACK_AND_MP
```

Comment:
Where is this defined, what does it mean?

Reply:
It is fine for me to keep it, if you expect to need it in the future.... if not, better remove to eliminate dead code.